ParquetSharp.Dataset/ParquetSharp.Dataset.csproj

```diff
@@ -2,10 +2,30 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>10.0</LangVersion>
+    <AssemblyName>ParquetSharp.Dataset</AssemblyName>
```

Comment:
I think that these are not necessary (if not provided then they are derived from a project name)

Reply:
Thanks, removed